task/thermal/src/control.rs

```diff
@@ -300,6 +312,7 @@ pub(crate) struct Max31790State {
 }
 
 impl Max31790State {
+    #[allow(dead_code)]
```

Comment:
should we maybe feature-flag the two different fan-controllers, instead of allowing dead code?

not a blocker either way, i imagine rustc eliminates the dead code?

Reply:
I decided to leave the dead code, because it's at least type-checked (and yeah `rustc` should eliminate it)